dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -245,6 +245,11 @@ DSRA4004.onconnect.sql=DSRA4004E: Unable to execute onConnect SQL command \"{0}\
 DSRA4004.onconnect.sql.explanation=An error occurred while attempting to execute the specified SQL command.
 DSRA4004.onconnect.sql.useraction=Refer to the exception message to help determine the cause of the problem.
 
+#4008			TRANSACTION_NONE_UNSUPPORTED
+TRANSACTION_NONE_UNSUPPORTED=DSRA4008: The JDBC driver does not support an isolation level of TRANSACTION_NONE.  The isolation level was set to TRANSACTION_NONE because of the data source configuration.
```

Comment:
TRANSACTION_NONE_UNSUPPORTED=DSRA4008: The JDBC driver does not support an isolation level of TRANSACTION_NONE.  **The data source configuration caused the isolation level to be set to TRANSACTION_NONE.**

Reply:
I don't like the language stating "caused the isolation level to be set to TRANSACTION_NONE".  The isolation level didn't get set to TRANSACTION_NONE - the attempt was unsuccessful, so this is misleading.  Recommend the following:
"The JDBC driver does not support an isolation level of TRANSACTION_NONE, which is configured on data source {0}."
where {0} is a substitution parameter for the data source identifier that the user ought to be looking at to fix the problem

Reply:
"The JDBC driver does not support an isolation level of TRANSACTION_NONE, which is configured on data source {0}." Looks good to me.

Reply:
@njr-11, This review corresponds to the line that it's commented on, line 249. We don't review the message prefixes.

Reply:
@lzdanski  - I don't understand your most recent comment above that mentions me.  I certainly wouldn't argue with what you stated, and I don't believe I wrote anything that contradicts it, so I'm wondering if you meant to place that comment elsewhere or direct it towards a different person?

Reply:
@njr-11 I think that there are multiple threads of comments about the same line. It's not clear what's a response to the message review comments vs. the code. Sorry about that!